### ISSUE TITLE
closure-size: qtbase:  attribute ‘libs’ missing, at ...pkgs/development/libraries/qt-5/5.4/qtbase/default.nix

### ISSUE THREAD

**Issue by deepfire** [NixOS/nixpkgs#12520]
While doing a `nix-env --upgrade --max-jobs 8 --cores 8 --no-build-output --leq '*' --show-trace` in latest `closure-size`, which has recent `master` and `staging` merged in:
 
```
[deepfire@andromedae:~]$ nix-env --upgrade --max-jobs 8 --cores 8 --no-build-output --leq '*' --show-trace
upgrading ‘firefox-42.0’ to ‘firefox-43.0.4’
upgrading ‘pciutils-3.3.1’ to ‘pciutils-3.4.1’
upgrading ‘gnupg-2.0.29’ to ‘gnupg-2.0.29’
upgrading ‘gpa-0.9.9’ to ‘gpa-0.9.9’
upgrading ‘isync-1.2.0’ to ‘isync-1.2.1’
upgrading ‘mu-0.9.12’ to ‘mu-0.9.13’
upgrading ‘sweethome3d-application-4.6.2’ to ‘sweethome3d-application-4.6.2’
upgrading ‘bootchart-0.14.7’ to ‘bootchart-0.14.7’
upgrading ‘ltrace-0.7.3’ to ‘ltrace-0.7.3’
upgrading ‘rlwrap-0.42’ to ‘rlwrap-0.42’
upgrading ‘morituri-0.2.3’ to ‘morituri-0.2.3.20151109’
upgrading ‘subversion-1.8.14’ to ‘subversion-1.9.3’
upgrading ‘git-2.5.0’ to ‘git-2.7.0’
upgrading ‘vue-3.3.0’ to ‘vue-3.3.0’
upgrading ‘lyx-2.1.4’ to ‘lyx-2.1.4’
upgrading ‘ethtool-4.0’ to ‘ethtool-4.0’
upgrading ‘gv-3.7.4’ to ‘gv-3.7.4’
upgrading ‘python-2.7.10’ to ‘python-2.7.11’
upgrading ‘samba-4.2.3’ to ‘samba-4.3.1’
upgrading ‘virt-manager-1.2.1’ to ‘virt-manager-1.3.1’
upgrading ‘qemu-2.4.0-x86-only’ to ‘qemu-2.4.1’
upgrading ‘emacs-24.5’ to ‘emacs-24.5’
upgrading ‘openssl-1.0.2d’ to ‘openssl-1.0.2e’
upgrading ‘cabal2nix-20150903’ to ‘cabal2nix-20151217’
upgrading ‘cmake-3.3.1’ to ‘cmake-3.4.0’
upgrading ‘picocom-1.8’ to ‘picocom-2.1’
upgrading ‘libreoffice-5.0.1.2’ to ‘libreoffice-5.0.4.2’
upgrading ‘python3-3.4.3’ to ‘python3-3.5.1’
upgrading ‘nix-1.10’ to ‘nix-1.10’
upgrading ‘nix-repl-1.10-8a2f5f0’ to ‘nix-repl-1.10-8a2f5f0’
upgrading ‘chromium-45.0.2454.85’ to ‘chromium-47.0.2526.106’
upgrading ‘links-1.00pre15’ to ‘links-1.00pre15’
upgrading ‘jwhois-4.0’ to ‘jwhois-4.0’
upgrading ‘unrar-5.2.7’ to ‘unrar-5.3.9’
upgrading ‘vim-7.4.827’ to ‘vim-7.4.827’
error: while trying to find an upgrade for ‘qtox-dev-20150821’:
while evaluating the attribute ‘nativeBuildInputs’ of the derivation ‘qtox-dev-20151221’ at /home/deepfire/src/nixpkgs/pkgs/applications/networking/instant-messengers/qtox/default.nix:11:3:
while evaluating the attribute ‘propagatedNativeBuildInputs’ of the derivation ‘qtbase-5.4.2’ at /home/deepfire/src/nixpkgs/pkgs/development/libraries/qt-5/5.4/qtbase/default.nix:33:3:
attribute ‘libs’ missing, at /home/deepfire/src/nixpkgs/pkgs/development/libraries/qt-5/5.4/qtbase/default.nix:179:33
```

**Comment by vcunat** [comment]
This is specific to the closure-size branch.

**Comment by vcunat** [comment]
These were fixed by 21bebe1592 and 2b76df28.
